staging/src/k8s.io/api/core/v1/resource.go

```diff
@@ -41,6 +41,14 @@ func (self *ResourceList) Memory() *resource.Quantity {
 	return &resource.Quantity{Format: resource.BinarySI}
 }
 
+// Returns the Storage request of VolumeClaimTemplates, or Storage Capacity of PersistentVolume.
```

Comment:
remove references to specific API types from the godoc

```suggestion
// Returns the Storage limit if specified.
```

Reply:
ok, ptal.

Reply:
ptal.